### PR TITLE
Add FoxeerH743 MPU6000

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-autopilots.rst [hunk 1]
```diff
@@ -90,6 +90,7 @@ Closed hardware
     AtomRC F405-NAVI <common-atomrcf405-navi>
     Emlid NAVIO2 (Linux) <common-navio2-overview>
     Flywoo F745 AIO BL_32/ Nano <common-flywoo-f745>
+    Foxeer H745 MPU600 <common-foxeerh743v1>
```
Comment (andyp1per) [inline common/source/docs/common-autopilots.rst:93]: 6000

File: common/source/docs/common-foxeerh743v1.rst [hunk 2]
```diff
@@ -0,0 +1,151 @@
+.. _common-foxeerH743v1:
+
+================
+Foxeer H7 MPU600
```
Comment (andyp1per) [inline common/source/docs/common-foxeerh743v1.rst:4]: 6000

File: common/source/docs/common-foxeerh743v1.rst [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+============
+
+`Foxeer <https://www.foxeer.com/foxeer-h7-mpu6000-fc-8s-dual-bec-barometer-x8-g-503>`__
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  STM32H743 ARM 480MHz)
```
Comment (andyp1per) [inline common/source/docs/common-foxeerh743v1.rst:21]: missing brace

File: common/source/docs/common-foxeerh743v1.rst [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+RC input is configured by default on the R1 (UART1_RX) pin. It supports all serial RC protocols. For protocols requiring separate half-duplex serial to transmit telemetry (such as FPort) you should setup SERIAL1 as an RC input serial port, with half-duplex, pin-swap and inversion enabled (:ref:`SERIAL1_OPTIONS<SERIAL1_OPTIONS>` = 15).
+
+The DJI connector also has UART4 RX input for DJI RC input. To use this you would need to set :ref:`SERIAL1_PROTOCOL<SERIAL1_PROTOCOL>` to something other than "23" (RCinput) and set this protocol for :ref:`SERIAL4_PROTOCOL<SERIAL4_PROTOCOL>` instead. 
+
+For PPM support on UART4_RX set :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` to 1 and use the UART4 RX pin as PPM (and any other serial RC protocol) input.
+
+OSD Support
+===========
+
+The SpeedyBeeF405Wing supports using its internal OSD using OSD_TYPE 1 (MAX7456 driver). External OSD support such as DJI or DisplayPort is supported using UART7 or any other free UART. See :ref:`common-msp-osd-overview-4.2` for more info.
```
Comment (andyp1per) [inline common/source/docs/common-foxeerh743v1.rst:99]: Foxeer!

**Comment by Hwurzburg** [comment]
thanks...

**Pull request merged**